extensions/wikia/ImageReview/ImageReviewHelper.class.php

```diff
@@ -488,13 +581,13 @@ public function getStatsData( $startYear, $startMonth, $startDay, $endYear, $end
 				while ( $row = $dbr->fetchObject( $res ) ) {
 					if ( !empty( $row->review_state ) ) {
 						if ( !isset( $data[ $reviewer ] ) ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS134)